tests/test_block_model.py

```diff
@@ -2,6 +2,9 @@
 from pytest import raises
 
 
+pytestmark = pytest.mark.tendermint
```

Comment:
I don't think we need to fix these tests for `Block` class it's going to be removed soon.